internal/service/streamprocessor/resource_test.go

```diff
@@ -0,0 +1,195 @@
+package streamprocessor_test
+
+import (
+	"context"
+	"fmt"
+	"regexp"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/hashicorp/terraform-plugin-testing/terraform"
+
+	"github.com/mongodb/terraform-provider-mongodbatlas/internal/common/conversion"
+	"github.com/mongodb/terraform-provider-mongodbatlas/internal/testutil/acc"
+)
+
+var (
+	resourceName = "mongodbatlas_stream_processor.processor"
+)
+
+func TestAccStreamProcessorRS_basic(t *testing.T) {
```

Comment:
nice. Can we have another `Acc` test using cluster source and testing the `change_stream_token`?

Reply:
Done!